ui-admin/src/study/kits/KitList.tsx

```diff
@@ -42,16 +105,43 @@ export default function KitList({ studyEnvContext }: { studyEnvContext: StudyEnv
     loadKits()
   }, [studyEnvContext.study.shortcode, studyEnvContext.currentEnv.environmentName])
 
+  const kitsByStatus = _groupBy(kits, kit => {
+    return statusTabs.find(tab => tab.status === kit.pepperStatus?.currentStatus)?.status || 'ERROR'
+  })
+
   return <LoadingSpinner isLoading={isLoading}>
-    <KitListView kits={kits} studyEnvContext={studyEnvContext}/>
+    <Tabs
+      activeKey={activeTab ?? undefined}
+      mountOnEnter
+      unmountOnExit
+      onSelect={setActiveTab}
+    >
+      { statusTabs.map(tab => {
```

Comment:
yeah, a big benefit of not using bootstrap tabs is getting to avoid the 'map' here.  You can just render the tab you want, which means you won't need to use the lodash groupby